docs/api-guide/authentication.md

```diff
@@ -217,38 +210,6 @@ It is also possible to create Tokens manually through admin interface. In case y
     TokenAdmin.raw_id_fields = ('user',)
 
 
-#### Schema migrations
```

Comment:
I was wondering too and assumed migrations are supposed to be used anyway.
I've just asked other opinions about that topic.